### PR TITLE
ROX-22625: Add tooltip for discovered cluster status and date

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

Rewrote tooltips:
* to remove reference to product branding
* in parallel form
* with input from Stephan about undetermined status ~and added link via info alert~

## Checklist
- [ ] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

Before you `yarn deploy` in ui folder:

```sh
export ROX_CLOUD_SOURCES=true
```

1. `yarn lint` in ui
2. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js 0 = 3951581 - 3951581
        total 1346 = 10544860 - 10543514
    * `ls -al build/static/js/*.js | wc`
        files 0 = 92 - 92
3. `yarn start` in ui

### Manual testing

Disregard info alert with link which has been deleted.

1. Visit /main/clusters/discovered-clusters with temporary data for request.

    * Secured
        Secured

    * Unsecured
        Unsecured

    * Undetermined

       Tooltip simplified to delete duplicated word and second sentence:
       > This cluster has been discovered by a cloud source, but has undetermined status because metadata collected from secured clusters does not guarantee a unique match.

        Undetermined

    * First discovered
        First_discovered

### Integration testing

Before you `yarn cypress-open` in ui/apps/platform folder:

```sh
export CYPRESS_ROX_CLOUD_SOURCES=true
```

* clusters/discovered-clusters/DiscoveredClustersPage.test.js

### CODE REVIEW

File: ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredCluster.tsx [hunk 1]
```diff
@@ -43,6 +43,17 @@ export function getStatusText(status: DiscoveredClusterStatus) {
     return statusMap[status] ?? statusMap.STATUS_UNSPECIFIED;
 }
 
+const statusTipMap: Record<DiscoveredClusterStatus, string> = {
+    STATUS_SECURED: 'This cluster is already secured.',
+    STATUS_UNSECURED: 'This cluster has been discovered by a cloud source, but is not yet secured.',
+    STATUS_UNSPECIFIED:
+        'This cluster has has been discovered by a cloud source, but has undetermined status because metadata collected from secured clusters does not guarantee a unique match. Sensor requires access to the AWS EC2 instance tags to determine the cluster state.',
```
Comment (stehessel) [inline ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredCluster.tsx:50]: So there are two primary situations in which unspecified status will occur:

1. EKS clusters not having EC2 instance tags.
2. Old sensor versions not providing enough metadata.

The AWS comment is only relevant to 1. I wonder how to communicate this properly without making it seem that in scenario 2, the AWS instance tags would help. Perhaps "On AWS Sensor requires access to the AWS EC2 instance tags to determine the cluster state."?

File: ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredClustersPage.tsx [hunk 2]
```diff
@@ -102,6 +103,21 @@ function DiscoveredClustersPage(): ReactElement {
...
                             Discovered clusters might not yet have secured cluster services.
                         </Text>
+                        <Alert
+                            variant="info"
+                            isInline
+                            title="Solution for discovered clusters that have undetermined status"
+                        >
+                            <ExternalLink>
+                                <a
+                                    href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#allow-access-to-tags-in-IMDS"
```
[the remark this answers is not in the document]
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredClustersPage.tsx]: Deleted link as discussed in chat.

It is questionable whether the address of docs link will be known within the next week.

File: ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredCluster.tsx (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredCluster.tsx]: Okay, as discussed with Stephan on chat:
1. Delete duplicated word in first sentence.
2. Delete second sentence and let docs explain the case analysis, which includes older versions of sensors.

Although input from docs team is always welcome, better to have sought that before now.

**Review by vjwilson — APPROVED** [review]

**Pull request merged**